secure_collaborative_computation_service/client_service/stub/chart/values.yaml

```diff
@@ -0,0 +1,7 @@
+service:
+  type: ClusterIP
+  port: 80
+
+deployment:
+  image: client-service:0.1.0
+  replicas: 1
```

Comment:
It is a good practice to have the `Resource Limits`

Reply:
I added resource limits and healtch checks to the deployment of the two services.